docs/oid-info.md

```diff
@@ -58,11 +60,89 @@ the git ref that the workflow run was based upon.
 This specifies the username identity in the OtherName Subject Alternative Name, as
 defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6).
 
+
+### 1.3.6.1.4.1.57264.1.8 | Source URI
+
+Should include a fully qualified source repository URL that the build was based
+on.
+
+### 1.3.6.1.4.1.57264.1.9 | Source Digest
+
+An immutable reference to a specific version of the source code that the build
+was based upon. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
+
+### 1.3.6.1.4.1.57264.1.10 | Source Ref
+
+The source ref that the build run was based upon. For example: `refs/head/main`.
+
+### 1.3.6.1.4.1.57264.1.11 | Source Identifier
+
+The stable identifier for the source repository the workflow was based upon.
+
+### 1.3.6.1.4.1.57264.1.12 | Source Owner URI
+
+Fully qualified URL for the owner of the source repository that the build was based
+on.
+
+### 1.3.6.1.4.1.57264.1.13 | Source Owner Identifier
+
+The stable identifier for the owner of the source repository that the workflow was based upon.
+
+### 1.3.6.1.4.1.57264.1.14 | Build Config URI
+
+A reference to the initiating build instructions.
+
+### 1.3.6.1.4.1.57264.1.15 | Build Config Digest
+
+An immutable reference to the specific version of the top-level build
+instructions. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
+
+### 1.3.6.1.4.1.57264.1.16 | Build Signer URI
+
+Reference to specific build instructions that are responsible for signing. Can be the same as Build Config URI. For example a reusable workflow in GitHub Actions or a Circle CI Orbs.
+
+Note this value would also be set in the Subject Alternative Name (SAN).
```

Comment:
For simplicity sake, let's just cut this line out. Even if it is the SAN, this value should still be filled out.

```suggestion
```